src/telliot_feeds/reporters/reporter_autopay_utils.py

```diff
@@ -308,14 +310,18 @@ async def get_feed_tip(query_id: bytes, autopay: TellorFlexAutopayContract) -> O
     try:
         single_query = {query_id: CATALOG_QUERY_IDS[query_id]}
     except KeyError:
-        if "TellorRNG" in str(query_id):
+        qtype_name, _ = decode_single("(string,bytes)", query_id)
```

Comment:
Ok, last thing is can you rename `query_id` to be query data, since we'd never be able to decode the query id into the type & parameter bytes? Can only do that from query data.

And if you absolutely need it to be called `query_id` for some reason, then put in a comment saying it's actually the query data